<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="de">
<context>
    <name>About</name>
    <message>
        <location filename="../about/about.py" line="35"/>
        <source>About {0} {1}</source>
        <translation>Über {0} {1}</translation>
    </message>
    <message>
        <location filename="../about/about.py" line="36"/>
        <source>Version: {0}</source>
        <translation>Version: {0}</translation>
    </message>
    <message>
        <location filename="../about/about.py" line="38"/>
        <source>Author</source>
        <translation>Autor</translation>
    </message>
    <message>
        <location filename="../about/about.py" line="39"/>
        <source>Contact</source>
        <translation>Kontakt</translation>
    </message>
    <message>
        <location filename="../about/about.py" line="40"/>
        <source>Change Log</source>
        <translation>Änderungen</translation>
    </message>
    <message>
        <location filename="../about/about.py" line="45"/>
        <source>&lt;p&gt;&lt;center&gt;&lt;b&gt;Author(s):&lt;/b&gt;&lt;/center&gt;&lt;/p&gt;</source>
        <translation>&lt;p&gt;&lt;center&gt;&lt;b&gt;Autor(en):&lt;/b&gt;&lt;/center&gt;&lt;/p&gt;</translation>
    </message>
    <message>
        <location filename="../about/about.py" line="46"/>
        <source>&lt;p&gt;{0}&lt;br&gt;</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../about/about.py" line="48"/>
        <source>Sourcepole AG - Linux &amp; Open Source Solutions
</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../about/about.py" line="49"/>
        <source>Weberstrasse 5, 8004 Zurich, Switzerland
</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../about/about.py" line="52"/>
        <source>Contact:
</source>
        <translation>Kontakt:
</translation>
    </message>
    <message>
        <location filename="../about/about.py" line="55"/>
        <source>Plugin Resources:
</source>
        <translation>Plugin-Ressourcen:
</translation>
    </message>
</context>
<context>
    <name>PGRasterImport</name>
    <message>
        <location filename="../pgraster_import.py" line="177"/>
        <source>&amp;PostGIS Raster Import</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../pgraster_import.py" line="164"/>
        <source>PostGIS Raster Import</source>
        <translation></translation>
    </message>
</context>
<context>
    <name>PGRasterImportDialog</name>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="235"/>
        <source>Error</source>
        <translation>Fehler</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="128"/>
        <source>No Database Connection Established.</source>
        <translation type="obsolete">Es wurde keine Datenbankverbindung hergestellt.</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="214"/>
        <source>Table exists</source>
        <translation>Die Tabelle existiert</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="214"/>
        <source>The selected table already exists in the database. Do you want to overwrite the table?</source>
        <translation>Die ausgewählte Tabelle existiert bereits in der Datenbank. Möchten Sie die Tabelle überschreiben?</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="58"/>
        <source>%s</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="147"/>
        <source>Username or password incorrect!</source>
        <translation>Falscher Benutzername oder falsches Passwort!</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="201"/>
        <source>PostGIS Raster Extension not installed in destination DB</source>
        <translation>Erweiterung „PostGIS Raster“ ist nicht in der Zieldatenbank installiert</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="233"/>
        <source>Success</source>
        <translation>Erfolg</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="233"/>
        <source>Raster successful uploaded to database</source>
        <translation>Raster wurde erfolgreich in die Datenbank geladen</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="235"/>
        <source>Upload failed</source>
        <translation>Hochladen gescheitert</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="337"/>
        <source>PostGIS Raster Import</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="229"/>
        <source>Upload cancelled</source>
        <translation>Upload abgebrochen</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="268"/>
        <source>Warning</source>
        <translation>Warnung</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="268"/>
        <source>Layers of type {0} are not supported!</source>
        <translation>Layer des Typs {0} werden nicht unterstützt!</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="337"/>
        <source>Could not load raster layer: database connection not available</source>
        <translation>Kann den Rasterlayer nicht laden: die Datenbankverbindung ist nicht verfügbar</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="158"/>
        <source>Cannot connect to {0}: Exception info: {1}</source>
        <translation>Keine Verbindung mit {0} möglich: Exception-Info: {1}</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.py" line="258"/>
        <source>Raster has no valid CRS</source>
        <translation>Raster hat kein gültiges KBS</translation>
    </message>
</context>
<context>
    <name>PGRasterImportDialogBase</name>
    <message>
        <location filename="../pgraster_import_dialog_base.ui" line="14"/>
        <source>PostGIS Raster Import</source>
        <translation></translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.ui" line="33"/>
        <source>Tablename:</source>
        <translation>Tabellenname:</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.ui" line="47"/>
        <source>Raster Layer:</source>
        <translation>Rasterebene:</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.ui" line="54"/>
        <source>DB-Connection:</source>
        <translation>DB-Verbindung:</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.ui" line="61"/>
        <source>Schema</source>
        <translation>Schema:</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.ui" line="97"/>
        <source>Upload</source>
        <translation>Hochladen</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.ui" line="110"/>
        <source>Close</source>
        <translation>Schließen</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.ui" line="130"/>
        <source>About</source>
        <translation>Über</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.ui" line="140"/>
        <source>Create Raster Overviews</source>
        <translatorcomment>besser: „Rasterübersichten (Pyramiden) erzeugen“, passte aber nicht ins Layout</translatorcomment>
        <translation>Übersichten erzeugen</translation>
    </message>
    <message>
        <location filename="../pgraster_import_dialog_base.ui" line="153"/>
        <source>Add Raster to canvas</source>
        <translatorcomment>besser: „Raster zur Karte hinzufügen“, passt aber nicht ins Layout</translatorcomment>
        <translation>zur Karte hinzufügen</translation>
    </message>
</context>
<context>
    <name>RasterUpload</name>
    <message>
        <location filename="../raster/raster_upload.py" line="102"/>
        <source>Creating table &apos;{table}&apos;...</source>
        <translation>Tabelle„{table}“ erstellen…</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="134"/>
        <source>Creating overview-level {level} for table &apos;{table}&apos;...</source>
        <translation>Übersichtsebene {level} für Tabelle „{table}“ anlegen…</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="149"/>
        <source>Registering raster columns of table &apos;%s&apos;...</source>
        <translation>Rasterspalten der Tabelle „%s“ werden registriert…</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="153"/>
        <source>Upload successful finished</source>
        <translation>Hochladen erfolgreich beendet</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="665"/>
        <source>Uploading tiles...</source>
        <translation>Kacheln hochladen…</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="727"/>
        <source>Calculating raster params for {sum_tiles} tiles ...</source>
        <translation>Berechnung von Rasterparametern für {sum_tiles}-Kacheln…</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="160"/>
        <source>Error</source>
        <translation>Fehler</translation>
    </message>
    <message>
        <location filename="../raster/raster_upload.py" line="160"/>
        <source>%s</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
